cmd/amp/storage.go

```diff
@@ -0,0 +1,222 @@
+	if err != nil {
+		return err
+	}
+	fmt.Println(reply.Val)
+	return nil
+}
+
+// storageGet validates the input command line arguments and retrieves storage key-value pair
+//by invoking the corresponding rpc/storage method
+func storageGet(amp *client.AMP, cmd *cobra.Command, args []string) (err error) {
+	if len(args) > 1 {
+		return errors.New("Too many arguments - check again!")
+	} else if len(args) == 0 {
+		return errors.New("Must specify storage key")
+	}
+	k := args[0]
+	if k == "" {
+		return errors.New("Must specify storage key")
+	}
+
+	request := &storage.GetStorage{Key: k}
+
+	client := storage.NewStorageClient(amp.Conn)
+	reply, err := client.Get(context.Background(), request)
+	if err != nil {
+		return err
+	}
+	fmt.Println(reply.Val)
+	return nil
+}
+
+// storageDelete validates the input command line arguments and deletes storage key-value pair
+// by invoking the corresponding rpc/storage method
+func storageDelete(amp *client.AMP, cmd *cobra.Command, args []string) (err error) {
+	if len(args) > 1 {
+		return errors.New("Too many arguments - check again!")
+	} else if len(args) == 0 {
+		return errors.New("Must specify storage key")
+	}
+	k := args[0]
+	if k == "" {
+		return errors.New("Must specify storage key")
+	}
+
+	request := &storage.DeleteStorage{Key: k}
+
+	client := storage.NewStorageClient(amp.Conn)
+	reply, err := client.Delete(context.Background(), request)
+	if err != nil {
+		return err
+	}
+	fmt.Println(reply.Val)
+	return nil
+}
+
+// storageUpdate validates the input command line arguments and updates storage key-value pair
+// by invoking the corresponding rpc/storage method
+func storageUpdate(amp *client.AMP, cmd *cobra.Command, args []string) (err error) {
+	if len(args) < 2 {
+		return errors.New("Must specify storage key/value - either or both of them are missing")
+	} else if len(args) > 2 {
+		return errors.New("Too many arguments - check again!")
+	}
+
+	k := args[0]
+	if k == "" {
+		return errors.New("Must specify storage key")
+	}
+
+	v := args[1]
+	if v == "" {
+		return errors.New("Must specify storage value")
+	}
+
+	request := &storage.UpdateStorage{Key: k, Val: v}
+
+	client := storage.NewStorageClient(amp.Conn)
+	reply, err := client.Update(context.Background(), request)
+	if err != nil {
+		return err
+	}
+	fmt.Println(reply.Val)
+	return nil
+}
+
+// storageList validates the input command line arguments and lists all the storage
+// key-value pairs by invoking the corresponding rpc/storage method
+func storageList(amp *client.AMP, cmd *cobra.Command, args []string) (err error) {
+	if len(args) > 0 {
+		return errors.New("Too many arguments - check again!")
+	}
+	request := &storage.ListStorage{}
+	client := storage.NewStorageClient(amp.Conn)
+	reply, err := client.List(context.Background(), request)
+	if err != nil {
+		return err
+	}
+	if reply == nil || len(reply.List) == 0 {
+		fmt.Println("No storage object is available")
+		return nil
+	}
+
+	//Format output
```

Comment:
@neha-viswanathan just nitpicking here, but can you use the table writer from the golang std library? You can see an example usage here: https://github.com/appcelerator/amp/blob/master/cmd/amp/topic-ls.go#L42-L47